api/controllers/stats.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const Trials = require('../models/trial');
+const bookshelf = require('../../config').bookshelf;
+const Promise = require('bluebird');
+
+function getTrialsCount() {
+  return Trials.count();
+}
+
+function getTrialsPerRegistry() {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('primary_register as registry'),
+      bookshelf.knex.raw('count(primary_register)::int')
+    )
+    .from('trialrecords').groupByRaw('primary_register')
+    .then((rows) => {
+      return rows;
+    });
```

Comment:
As far as I know, this `.then((rows) => { return rows; });` isn't doing anything, so it can be removed safely.